lib/gyp/generator/ninja/index.js

```diff
@@ -350,8 +353,8 @@ Ninja.prototype.actionCmd = function actionCmd(base, toBase, cmds) {
   if (this.flavor !== 'win32')
     return res;
 
-  // TODO(indutny): escape quotes in res
-  return `cmd.exe /s /c "${res}"`;
+  const nw = gyp.platform.win.getNinjaWrapper(this.targetArch);
+  return `${nw} cmd.exe /s /c "${res}"`;
```

Comment:
I'm not sure that's well formed (in the sense pasting `gaga" "lala` in a windows console works), but if it is, it'll work.
Thank god it's not [VBScript](http://stackoverflow.com/questions/2942554/adding-quotes-to-a-string-in-vbscript)